api/server/models/meetingRequestModel.js

```diff
@@ -1,21 +1,25 @@
-// var mongoose = require("mongoose");
+var mongoose = require("mongoose");
 
-// var Option = require("./optionModel");
+var Option = require("./optionModel");
 
-// var meetingRequestSchema = mongoose.Schema({
-//   author: mongoose.Types.ObjectId,
-//   deadline: { type: Date, required: true },
-//   groupID: mongoose.Types.ObjectId,
-//   multiPoll: [Option]
-//   uniPoll: Option,
-//   status: Number,
-// });
+var meetingRequestSchema = mongoose.Schema({
+  author: {type : mongoose.Types.ObjectId, required: true},
+  deadline: { type: Date, required: true },
+  groupID: {type : mongoose.Types.ObjectId, required: true},
+  multiPoll: [Option],
```

Comment:
As I mentioned above, you can probably just merge these two ('multiPoll' and 'uniPoll') into one attribute called 'options' that is just an array of Option's.

Reply:
I think that's a good idea